platform/macos/export/lipo.cpp

```diff
@@ -32,8 +32,6 @@
 
 #include "lipo.h"
 
-#ifdef MODULE_REGEX_ENABLED
```

Comment:
You can also remove the include for `modules_enabled.gen.h` in those files if not used.

Reply:
Done.